desk/app/profile.hoon

```diff
@@ -134,6 +137,12 @@
     ::
     :-  [%pass /eyre/cache %arvo %e %set-response '/profile' `[| %payload payload]]
     (spout:rudder id payload)
+  ?:  =('/expose/style/shared.css' url.request)
+      :-  [200 ['content-type' 'text/css']~]
+      `(as-octs:mimes:html style-shared)
```

Comment:
The %profile agent only serves requests for `/profile` and stuff underneath that path, so this is never going to get hit. %expose will handle it though.

Reply:
Removed!